### ISSUE TITLE
Infinite Loop on calling ReverseMd5 method 

### ISSUE THREAD

**Issue by MUHAMMADTAYYAB** [ArtskydJ/reverse-md5#14]
Hi, I am facing an infinite loop issue on calling the following method and passing an md5 hash to it.

let plainString = ReverseMd5()('cfbf799328dd0ae4b223099404b8bde9'); 

I am unable to get control of my code after calling that function. Socket hangs on

**Comment by ArtskydJ** [comment]
TL;DR, this code is really slow, and should not be used seriously.  It was one of my first javascript projects, when I was learning the language in mid 2014.

---------

If you don't pass in any options, ReverseMd5 will attempt to brute force an md5 hash for roughly 5 trillion strings.

Even if your computer is somehow calculating a billion hashes per second, it would take well over an hour to stop trying any more.  I expect that passing a hash for a string that is longer than 5 characters is infeasible for this code to do.

There are serious tools like hashcat that you might find more useful.

------------

Or am I misunderstanding you, and there's actually an infinite loop bug?  If that's the case, feel free to re-open this issue.
